slave/process-users-emails/changelog

```diff
@@ -0,0 +1,5 @@
+perun-slave-process-users-emails (3.1.1) stable; urgency=low
```

Comment:
I know it technically doesn't matter, but shouldn't we start with something like `3.0.0` or `1.0.0`?